kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionContainerGrid.java

```diff
@@ -78,6 +79,8 @@
 
     private ExpressionContainerUIModelMapper uiModelMapper;
 
+    private Optional<Consumer> onUndoClear;
```

Comment:
For same reason of https://github.com/kiegroup/kogito-editors-java/pull/175/files#r768742951, I would avoid to use an Optional as class field. My suggestion is to remove the Optional in this point, and return it in `getUndoClear()` method, returning it in this way `return Optional.ofNullable(onUndoClear);`

Reply:
I don't understand why this is an issue.